charts/gardener/controlplane/charts/runtime/templates/apiserver/hvpa.yaml

```
@@ -0,0 +1,73 @@
+{{- if .Values.global.apiserver.hvpa.enabled }}
```

```suggestion
{{- if and .Values.global.apiserver.enabled .Values.global.apiserver.hvpa.enabled }}
```
I think, it only makes sense to deploy the HVPA if the apiserver itself is enabled.

---

Sure. Will add the labels to `vpa` chart too